CHANGELOG.md

```diff
@@ -15,7 +15,19 @@ it according to semantic versioning. For example, if your PR adds a breaking cha
 should change the heading of the (upcoming) version to include a major version bump.
 
 -->
-# v4.0.1 (upcoming)
+# v4.0.2 (upcoming)
+
+# v4.0.1
+
+## @rjsf/core
+- Added new `_internalFormWrapper` prop to `Form` that supports the `semantic-ui` and `material-ui` themes to work when `tagName` is provided
```

Comment:
Can we update the changelog not to mention `_internalFormWrapper`? Maybe we just say something like "Fixed a regression in which tagName did not work with material-ui 4 theme"